misc/deprecated/domain/api/atrium-domain-api-common/src/main/kotlin/ch/tutteli/atrium/domain/creating/charsequence/contains/creators/CharSequenceContainsAssertions.kt

```diff
@@ -32,11 +32,25 @@ interface CharSequenceContainsAssertions {
         expected: List<CharSequenceOrNumberOrChar>
     ): AssertionGroup
 
+    //TODO remove with 1.0.0
+    @Deprecated("Use AssertImpl.charSequence.contains.defaultTranslationOf; will be removed with 1.0.0",
+        ReplaceWith(
+            "AssertImpl.charSequence.contains.defaultTranslationOf(checker, expected, *otherExpected)",
+            "ch.tutteli.atrium.creating.AssertImpl"
+        )
+    )
```

Comment:
You suggest to replace it with the same thing :wink:
I see that the description of the task is no longer up-to-date. We don't plan to provide a replacement, thus change to:
```suggestion
    @Deprecated("Will be removed with 1.0.0")
```
Same for other places